src/sqlfluff/core/templaters/base.py

```diff
@@ -59,6 +59,7 @@ class RawFileSlice(NamedTuple):
     raw: str
     slice_type: str
     source_idx: int
+    slice_subtype: Optional[str] = None
```

Comment:
Either `"loop"` or `None`. We use this in the heuristic to detect bad patches to a templated loop body.